packages/release-mastermind/src/utils/parsingData.ts

```diff
@@ -1,16 +1,18 @@
+import { Label, Labels } from "../../types";
```

Comment:
Codacy has a fix for the issue: [There should be no space after '{'.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
import {Label, Labels } from "../../types";
```